organization/app/organization/[id]/layout.tsx

```
@@ -1,12 +1,326 @@
+                        const isActive = pathname === item.path;
+                        return (
+                          <li key={item.name}>
+                            <Link href={item.path} passHref>
+                              <div
+                                className={`flex items-center px-3 py-3 rounded-xl cursor-pointer transition-all ${
+                                  isActive 
+                                    ? "bg-blue-50 text-blue-600" 
+                                    : "text-gray-600 hover:bg-gray-100"
+                                }`}
+                                onClick={() => setMobileMenuOpen(false)}
+                              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)